modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSActionDropdownItem.java

```diff
@@ -7,13 +7,27 @@
 
 import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
 
+import java.util.List;
 import java.util.Map;
 
 /**
  * @author Marco Leo
  */
 public class FDSActionDropdownItem extends DropdownItem {
 
+	public FDSActionDropdownItem() {
+	}
```

Comment:
Is this needed?

Reply:
Yes, we needed in [/FDSActionDropdownItemBuilder class](https://github.com/antonio-ortega/liferay-portal/blob/LPD-70100-and-builder/modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSActionDropdownItemBuilder.java#L1041-L1042)